lib/workable/client.rb

```diff
@@ -257,7 +258,8 @@ def create_rating(candidate_id, member_id, comment, score)
       end
     end
 
-    def events(params = {})
+    def events(params: {}, context: DEFAULT_EVENTS_CONTEXT)
```

Comment:
With this defaultValue aren't we always sending the context parameter?
Isn't it unnecessary for the workable API key integration?

Reply:
If there is no effect I would say to send it anyway. Approved